### DISCUSSION TITLE
Data subscription with multiple databases

### DISCUSSION THREAD

**Discussion by marcuslindblom** [ravendb/ravendb#18449] [Q&A]
My setup consists of a workspace with multiple teams, each team with it's own database. A workspace often has more than one team. How can I setup subscriptions across multiple databases so that the user can receive changes from all teams in the workspace?

**Comment by ayende** [comment]
Subscription is per database.
You need to register the subscription on all databases and connect to all of them.

**Reply by ayende** [reply]
Here is how you open a subscription in a non-default database

```
ds1.Subscriptions.GetSubscriptionWorker(new SubscriptionWorkerOptions(), database: "my-db")
```
